ibis/file/csv/api.py

```diff
@@ -0,0 +1,5 @@
+from ibis.file.csv.client import CSVClient
+
+
+def connect(dictionary):
```

Comment:
Needs a small docstring describing what `dictionary` consists of since this is a user-facing API.

Reply:
done